src/App.tsx

```diff
@@ -1,16 +1,27 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [currentMoviesList,
+    setCurrentMoviesList] = useState([...moviesFromServer]);
```

Comment:
You don't have to copy it, you use it only as initial value, so you won't have any problem with referencing it later
```suggestion
    setCurrentMoviesList] = useState(moviesFromServer);
```